### PR TITLE
modify_table creation queries

### PR DESCRIPTION
**Pull request by Thumimku**
### Proposed changes in this pull request

Modify remote fetch table columns for remote resource URI and error status

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mssql.sql [hunk 1]
```diff
@@ -1152,8 +1152,10 @@ CREATE TABLE IDN_REMOTE_FETCH_CONFIG (
   ACTION_LISTENER_TYPE VARCHAR(255) NOT NULL,
   CONFIG_DEPLOYER_TYPE VARCHAR(255) NOT NULL,
   REMOTE_FETCH_NAME VARCHAR(255),
+  REMOTE_RESOURCE_URI VARCHAR(255) NOT NULL,
   ATTRIBUTES_JSON TEXT NOT NULL,
-  PRIMARY KEY (ID)
+  PRIMARY KEY (ID),
+  CONSTRAINT UC_REMOTE_RESOURCE_URI UNIQUE (REMOTE_RESOURCE_URI)
```
Comment (ruwanta) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mssql.sql:1158]: uniqueness need to be Tennat + ResourceURI +branch

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql [hunk 2]
```diff
@@ -1535,13 +1535,15 @@ IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE RESTRICT
 CREATE TABLE IDN_REMOTE_FETCH_CONFIG(
 	ID VARCHAR(255) NOT NULL,
     TENANT_ID INTEGER NOT NULL,
-    IS_ENABLED CHAR(1) NOT NULL,
+	IS_ENABLED CHAR(1) NOT NULL,
```
Comment (nilasini) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1538]: formatting issue

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql [hunk 3]
```diff
@@ -1564,12 +1566,14 @@ CREATE TABLE IDN_REMOTE_FETCH_REVISIONS(
...
 	DEPLOYMENT_STATUS VARCHAR2(255),
 	ITEM_NAME VARCHAR2(255),
+	DEPLOY_ERR_LOG CLOB,
 	CONSTRAINT PK_IDN_REMOTE_FETCH_REVISIONS PRIMARY KEY (ID),
 	FOREIGN KEY (CONFIG_ID) REFERENCES IDN_REMOTE_FETCH_CONFIG (ID) ON DELETE CASCADE,
 	CONSTRAINT UC_REVISIONS UNIQUE (CONFIG_ID, ITEM_NAME)
 )/
 
 
+
```
Comment (nilasini) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql:1576]: remove extra new line

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql (hunk 2 above)
Comment (Thumimku) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql:1538]: Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql (hunk 3 above)
Comment (Thumimku) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql:1576]: Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143

**Pull request merged**